homeassistant/components/whois/sensor.py

```diff
@@ -48,7 +50,7 @@
 class WhoisSensorEntityDescriptionMixin:
     """Mixin for required keys."""
 
-    value_fn: Callable[[Domain], int | None]
+    value_fn: Callable[[Domain], datetime | int | str | None]
```

Comment:
`datetime | StateType`?

Reply:
StateType is broader, this is more specific

Reply:
Only by one (float), this is fine as is :+1:

Reply:
yes, but I don't currently expect it to return a float. If it does, that would be unexpected / changed behavior :D